Fixed issues with handling null values. (#27031)
demisto/content: Packs/FiltersAndTransformers/ReleaseNotes/1_2_19.md

#### Scripts

##### LastArrayElement
- Updated the Docker image to: *demisto/python3:3.10.11.61265*.

> - Fixed the way in which the script handled null value given to the argument parameters, instead of returning 'None', it will return a meaningful error message. 

Reviewer: ```suggestion
- Fixed the way in which the script handled a null value given to the argument parameters. Instead of returning 'None', it will return a meaningful error message. 
```

##### FirstArrayElement
- Updated the Docker image to: *demisto/python3:3.10.11.61265*.

> - Fixed an issue with handling null value given to the argument parameters. 

Reviewer 2: ```suggestion
- Fixed the way in which the script handled null value given to the argument parameters, instead of returning 'None', it will return a meaningful error message. 
```

> - Fixed the way in which the script handled null value given to the argument parameters, instead of returning 'None', it will return a meaningful error message.

Reviewer: ```suggestion
- Fixed the way in which the script handled a null value given to the argument parameters. Instead of returning 'None', it will return a meaningful error message.
```
